### PR TITLE
Travis config for Storage and DB integration tests

### PR DESCRIPTION
**Pull request by paulb777**
Travis has a [way to use encrypted files](https://docs.travis-ci.com/user/encrypting-files/).  This enables us to run integration tests in addition to unit tests.

The decryption is not supported on PRs, so the integration tests will only run after the code is merged to master.

It's initially implemented for FirebaseStorage and FirebaseDatabase integration tests.

The urgency was bumped up after we had a FirebaseStorage bug require a release respin that would have been caught by integration tests.

### CODE REVIEW

File: scripts/install_prereqs.sh [hunk 1]
```diff
@@ -27,6 +27,14 @@ case "$PROJECT-$PLATFORM-$METHOD" in
     bundle exec pod install --project-directory=Example --repo-update
     bundle exec pod install --project-directory=Functions/Example
     bundle exec pod install --project-directory=GoogleUtilities/Example
+
+    # Set up GoogleService-Info.plist for Storage and Database integration tests. The decrypting
+    # is not supported for pull requests. See https://docs.travis-ci.com/user/encrypting-files/
+    if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+        openssl aes-256-cbc -K $encrypted_2c8d10c8cc1d_key -iv $encrypted_2c8d10c8cc1d_iv \
+            -in GoogleService-Info.plist.enc -out Example/Storage/App/GoogleService-Info.plist -d
```
Comment (wilhuff) [inline scripts/install_prereqs.sh]: Since this is only useful for Travis, could we put the `GoogleService-Info.plist.enc` somewhere other than the root?

**Comment by paulb777** [comment]
Travis still works on PRs. 

I'll need an approval and merge to confirm that the integration tests run correctly after the merge.

**Comment by paulb777** [comment]
Ready for review

**Comment by paulb777** [comment]
cc: @schmidt-sebastian @morganchen12

**Pull request merged**